src/modules/module_utils.c

```diff
@@ -413,6 +422,54 @@ void module_strip_punctuation_default(char *buf)
 	module_strip_punctuation_some(buf, "~#$%^&*+=|<>[]_");
 }
 
+gchar *module_multicases_string_replace(
+	const gchar *text,
+	const MulticasesString *mcstr)
+{
+	GRegex *regex;
+	GError *error = NULL;
+	gchar *result;
+
+	result = g_strdup(text);
```

Comment:
You don't need to `strdup` `text`. In the error case, you can just return `text`. In the success case, you will return the result.